build.psm1

```diff
@@ -750,8 +750,8 @@ function New-PSOptions {
         [ValidateSet("Debug", "Release", "CodeCoverage", '')]
         [string]$Configuration,
 
-        [ValidateSet("net5.0")]
-        [string]$Framework = "net5.0",
+        [ValidateSet("net6.0")]
+        [string]$Framework = "net6.0",
```

Comment:
Is this done by the `udpatedotnetruntime.ps1`, or manually?

Reply:
fived them in second commit. the script does not change this.